ext/exif/exif.c

```diff
@@ -4039,7 +4058,7 @@ static bool exif_process_IFD_in_TIFF_impl(image_info_type *ImageInfo, size_t dir
 		exif_error_docref(NULL EXIFERR_CC, ImageInfo, E_NOTICE, "Read from TIFF: filesize(x%04X), IFD dir(x%04X + x%04X)", ImageInfo->FileSize, dir_offset, 2);
 #endif
 		php_stream_seek(ImageInfo->infile, dir_offset, SEEK_SET); /* we do not know the order of sections */
-		php_stream_read(ImageInfo->infile, (char*)ImageInfo->file.list[sn].data, 2);
+		read_from_stream_file_looped(ImageInfo->infile, (char*)ImageInfo->file.list[sn].data, 2);
```

Comment:
I know this is not done in the original call but shouldn't this check if there is an error?